### DISCUSSION TITLE
Binary Eye is great!

### DISCUSSION THREAD

**Discussion by enewmen** [community/community#34929] [Other Feature Feedback, Questions, & Ideas]
Just giving feedback.
After using lots of bloated barcode readers with ads,  Binary Eye is smaller,  faster, minimal, better - by a large margin.  Open source on F Droid also means its free from viruses and spyware. This is now the only barcode reader I'll consider. I hope the development continues.

**Comment by MelphinDev** [comment]
This seems off-topic.

**Reply by enewmen** [reply]
On topic. But I had trouble adding feedback to F Droid and APK Pure. ANYWAY, Just accept a compliment.
